src/interactive.js

```diff
@@ -0,0 +1,89 @@
+import { getStorage, setStorage } from './storage.js';
+
+const target = document.getElementById('todo-tasks');
+target.classList.add('todo-tasks');
+const items = target.getElementsByTagName('li');
+let current = null;
+
+export function dragStart() {
+  current = this;
+  for (let b = 0; b < items.length; b += 1) {
+    const it = items[b];
+    // apply this style to list except the darraged one;
+    // if (it !== current) {
+    //   it.classList.add('hint');
+    // }
```

Comment:
## optional

- [ ] Try to delete the comments in `.js` and `.css` files. It will help you to make the code well organized.